awesomeCreateApp.js

```diff
@@ -15,8 +13,9 @@ const terminalOpts = {
 }
 
 const createApp = folderName => {
-  spawn('create-react-app', [folderName], terminalOpts)
-    .then( () => showQuestions(folderName))
+  spawnPromise('create-react-app', [folderName], terminalOpts)
+    .then(() => showQuestions(folderName))
+    .catch((error) => console.log('create app rejected', error))
```

Comment:
I think add error handling is not related with this particular PR, we could open an issue for that